virtualizarr/xarray.py

```diff
@@ -265,7 +281,7 @@ def virtual_vars_from_kerchunk_refs(
     refs: KerchunkStoreRefs,
     drop_variables: list[str] | None = None,
     virtual_array_class=ManifestArray,
-) -> Mapping[str, xr.Variable]:
+) -> dict[str, xr.Variable]:
```

Comment:
For these, it's probably better to mark these as returning a `dict` (since that's what we return).

In general, I think the guidance is to be broad with what you accept (so something like a `Mapping`) and narrow in what you return.